quipucords/tests/scanner/openshift/test_api.py

```diff
@@ -440,3 +456,35 @@ def test_retrieve_workloads(ocp_client: OpenShiftApi):
     assert_elements_type(workloads, OCPWorkload)
     assert len(workloads) < len(pods)
     assert {p.app_name for p in pods} == {a.name for a in workloads}
+
+
+@pytest.fixture
+def node_metrics_query():
+    """Return a simple node metrics query."""
+    return "count by(instance) (max by(node, instance) (cluster:cpu_core_node_labels))"
+
+
+@pytest.mark.vcr_primer(VCRCassettes.OCP_METRICS_CACHE)
+@pytest.mark.vcr(
+    VCRCassettes.OCP_ROUTE,
+    VCRCassettes.OCP_METRICS_CACHE,
```

Comment:
please add VCRCassettes.OCP_DISCOVERER_CACHE and recreate the cassette

Reply:
worked like a champ, Thanks. https://github.com/quipucords/quipucords/pull/2438/commits/931cc883819bb47e4f2c50a279c40b04e1c01156.